playbooks/roles/user/templates/default.bashrc.j2

```diff
@@ -56,16 +56,16 @@ fi
 command -v ec2metadata >/dev/null 2>&1 && {  INSTANCEID=$(ec2metadata --instance-id); }
 
 if [ "$color_prompt" = yes ]; then
-    PS1='{{ USER_CMD_PROMPT }}${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h $INSTANCEID\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+    PS1='{{ USER_CMD_PROMPT }}${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\H $INSTANCEID\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
```

Comment:
Use `\H` instead of `\h`, per `man bash`:
```
\h     the hostname up to the first `.'
\H     the hostname
```

This gives us hostnames on the shell prompt like
- `rabbitmq1.stage.vpc`
instead of
- `rabbitmq1`